frontend/app/src/components/AssetBalances.vue

```diff
@@ -11,14 +14,20 @@ const props = withDefaults(
     loading?: boolean;
     hideTotal?: boolean;
     hideBreakdown?: boolean;
+    stickyHeader?: boolean;
   }>(),
   {
     loading: false,
     hideTotal: false,
-    hideBreakdown: false
+    hideBreakdown: false,
+    stickyHeader: false
   }
 );
 
+defineComponent({
```

Comment:
If it is only the name of the component we need to set you can use defineOptions({ name: }) instead.